_posts/2020-08-03-roadmap.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Roadmap
+---
+
+On June 18 2020, MDAnalysis was pleased to release the first major version, 1.0.0. As described in our [2019 roadmap](https://www.mdanalysis.org/2019/11/06/roadmap/), this is the last version that supports 2.7. We will continue backporting relevant bug fixes where feasible (e.g. the upcoming 1.0.1), but the next major release will be 2.0.0, which will support Python 3.6+.
```

Comment:
Btw, we never announced 1.0.0 because we're waiting for 1.0.1 to get out to fix the somewhat embarrassing selection slow-down... :-p

Reply:
Yeah... should I take that out? Might be hard to count number of formats in 1.0.0 without mentioning it in any way...